crypto/evp_extra/internal.h

```diff
@@ -26,18 +26,25 @@ typedef struct {
   uint8_t has_private;
 } KYBER512_KEY;
 
+typedef struct {
+  uint8_t pub[1952];
+  uint8_t priv[4000];
+  char has_private;
+} DILITHIUM3_KEY;
+
 extern const EVP_PKEY_ASN1_METHOD dsa_asn1_meth;
 extern const EVP_PKEY_ASN1_METHOD ec_asn1_meth;
 extern const EVP_PKEY_ASN1_METHOD rsa_asn1_meth;
 extern const EVP_PKEY_ASN1_METHOD rsa_pss_asn1_meth;
 extern const EVP_PKEY_ASN1_METHOD ed25519_asn1_meth;
 extern const EVP_PKEY_ASN1_METHOD x25519_asn1_meth;
 extern const EVP_PKEY_ASN1_METHOD kyber512_asn1_meth;
+extern const EVP_PKEY_ASN1_METHOD dilithium3_asn1_meth;
 
 extern const EVP_PKEY_METHOD ed25519_pkey_meth;
 extern const EVP_PKEY_METHOD x25519_pkey_meth;
 extern const EVP_PKEY_METHOD kyber512_pkey_meth;
-extern const EVP_PKEY_METHOD hkdf_pkey_meth;
```

Comment:
why is this line removed?

Reply:
This likely came from when I merged main. Why is it included in this list, if it isn't in [p_methods.c](https://github.com/awslabs/aws-lc/blob/main/crypto/evp_extra/p_methods.c#L10-L14)?

Reply:
I guess it comes in to play [here](https://github.com/awslabs/aws-lc/blob/main/crypto/fipsmodule/evp/internal.h#L270)